### ISSUE TITLE
How does this plugin improve accessibility ?

### ISSUE THREAD

**Issue by arobase-che** [florianeckerstorfer/remark-a11y-emoji#145]
Hello,

I would like to know how does that plugin is adding anything to the UX of users using assistive technologies?

I always believed that [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) should be used on icons (or icon-like images as small SVG). But emojis are not just plain images and should already have a description based on the Unicode description and CLDR name (which are in english but should be translated to user language I guess).

I'm don't understand so why this plugin is necessary.

I'm pretty sure, there is a good explanation of why this plugin is necessary and maybe this explanation should be in the README.md of the project for people like me who don't have a good enough accessibility background. 

Thank you!

**Comment by florianeckerstorfer** [comment]
Hello and thank you for the question. I don't know if anything has changed in this since I created this plugin 2020, but not every screenreader supports emoji. At least some ignore emoji correctly if they are not [manually marked up as images](https://www.w3.org/WAI/WCAG21/Techniques/html/H86).

I appreciate the hint that this should be much clearer in the README. Using such an automatic fallback for description should be a last resort, since the unicode description is not always the best description. Consider that an eggplant does not always mean eggplant.

**Comment by arobase-che** [comment]
Thank you very much for the explanation. :)
